### PR TITLE
fixes #12264 - regenerating .rubocop_todo.yml to remove warnings

### PR DESCRIPTION
**Pull request by komidore64**
Foreman has Rubocop's version pinned to 0.28.0. At some point
it seems that we generated a `.rubocop_todo.yml` for 0.30.x which has a
different set of cops than 0.28.0. This was causing our Rubocop runs to
throw warnings about unknown cops. It hasn't caused any problems, other
than simply being annoying.

In this commit, I've regenerated `.rubocop_todo.yml` so that it's been
generated by 0.28.0. This should remove those pesky 'unrecognized cops'
errors.

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -1,148 +1,78 @@
...
-# Offense count: 15
+# Offense count: 12
 # Cop supports --auto-correct.
 Lint/StringConversionInInterpolation:
   Enabled: false
 
-# Offense count: 586
+# Offense count: 1
+Lint/UselessAssignment:
+  Enabled: false
```
Comment (daviddavis) [inline .rubocop_todo.yml]: I probably would fix the one error rather than disable this cop.

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -1,148 +1,78 @@
...
+# Offense count: 98
 Metrics/CyclomaticComplexity:
   Max: 25
 
-# Offense count: 6007
+# Offense count: 5618
 # Configuration parameters: AllowURI, URISchemes.
 Metrics/LineLength:
-  Max: 200
+  Max: 202
```
Comment (daviddavis) [inline .rubocop_todo.yml]: 200 > 202

File: .rubocop_todo.yml [hunk 3]
```diff
@@ -1,148 +1,78 @@
...
-# Offense count: 1
-# Cop supports --auto-correct.
-# Configuration parameters: Include.
-Rails/FindEach:
-  Enabled: false
-
-# Offense count: 66
 # Configuration parameters: EnforcedStyle, SupportedStyles.
-Rails/TimeZone:
+Style/BracesAroundHashParameters:
```
Comment (daviddavis) [inline .rubocop_todo.yml]: Would be better to fix this.

File: .rubocop_todo.yml [hunk 4]
```diff
@@ -1,148 +1,78 @@
...
-# Configuration parameters: Exclude.
-Style/FileName:
-  Enabled: false
-
-# Offense count: 37
-# Cop supports --auto-correct.
-# Configuration parameters: EnforcedStyle, SupportedStyles.
-Style/FirstParameterIndentation:
+# Configuration parameters: Keywords.
+Style/CommentAnnotation:
```
Comment (daviddavis) [inline .rubocop_todo.yml]: Only 1 offense: probably should fix this.

**Comment by daviddavis** [comment]
It also looks like `Style/SpaceAroundOperators` was disabled. Would be nice to reenable that.

https://github.com/Katello/katello/commit/3e035bf0e6d7623a825bbdf586f330f5021c8e3f#diff-74423563ec7c74f4cb7f74689b49e7c1R137

**Comment by komidore64** [comment]
@daviddavis okay cool. i'll address your comments. thanks!

**Comment by daviddavis** [comment]
APJ + commit squash.

**Comment by komidore64** [comment]
@daviddavis squashed

**Comment by daviddavis** [comment]
ACK

**Pull request merged**